WordPress/gutenberg
# Framework: Use @wordpress/is-shallow-equal for shallow equality

This pull request seeks to incorporate the new [`@wordpress/is-shallow-equal`](https://www.npmjs.com/package/@wordpress/is-shallow-equal) package as a replacement for shallow equality. See also https://github.com/WordPress/packages/pull/110 and the module README.md for the _raison d'être_ of the package. 

Plugins may also depend on the `wp-is-shallow-equal` script handle and reference the utility via the global `wp.isShallowEqual`.

__Testing instructions:__

Ensure unit tests pass:

```
npm run test-unit
```

Verify that no errors occur in the load or basic usage of the Gutenberg editor.
